src/pages/index.tsx

```diff
@@ -2,10 +2,12 @@ import * as React from 'react';
 import { Route, Switch } from 'react-router-dom';
 
 import Home from './home';
+import WorkAccount from './WorkAccount';
```

Comment:
you can just call this `Account`; it's in a separate dir from the other components and it'll make it easier once it port it.

Reply:
👍